New site preview (#75)
common-workflow-language/cwl-website: content/community.md

---
layout: page
permalink: /community/
title: Community 
class: page-short
---

<figure class="float-right"><a href="https://open-stand.org/about-us/principles" class="open-stand"><picture><source srcset="https://github.com/common-workflow-language/cwl-website/raw/main/openstand-128x128-blue.webp" type="image/webp"><img src="./Common Workflow Language_files/openstand-128x128-blue.png" alt="Open Stand badge - the modern paradigm for standards" width="128" height="128"></picture></a></figure>

> CWL is developed by a multi-vendor working group consisting of organizations and individuals aiming to enable scientists to share data analysis workflows. The CWL project is maintained on Github and we follow the Open-Stand.org principles for collaborative open standards development. Legally CWL is a member project of Software Freedom Conservancy and is formally managed by the elected CWL leadership team, however every-day project decisions are made by the CWL community which is open for participation by anyone.

Reviewer: ```suggestion
CWL is developed by a multi-vendor working group consisting of organizations and individuals aiming to enable scientists to share data analysis workflows. The CWL project is maintained on Github and we follow the [Open-Stand.org principles](https://open-stand.org/about-us/principles/) for collaborative open standards development. Legally CWL is a member project of [Software Freedom Conservancy](https://sfconservancy.org/) and is formally managed by the elected CWL leadership team, however everyday project decisions are made by the CWL community which is open for participation by anyone.
```

> CWL is developed by a multi-vendor working group consisting of organizations and individuals aiming to enable scientists to share data analysis workflows. The CWL project is maintained on Github and we follow the Open-Stand.org principles for collaborative open standards development. Legally CWL is a member project of Software Freedom Conservancy and is formally managed by the elected CWL leadership team, however every-day project decisions are made by the CWL community which is open for participation by anyone.

Author: ```suggestion
CWL is developed by a multi-vendor working group consisting of organizations and individuals aiming to enable scientists to share data analysis workflows.
The CWL project is [maintained on Github](https://github.com/common-workflow-language/) and we follow the [Open-Stand.org principles for collaborative open standards development](https://open-stand.org/about-us/principles/). Legally CWL is a member project of [Software Freedom Conservancy](https://sfconservancy.org/news/2018/apr/11/cwl-new-member-project/)
and is formally managed by the elected [CWL leadership team](/governance/#cwl-leadership-team), however every-day project decisions are made by the CWL community which is open for participation by anyone.
```

{% include gitter.html %}

{:.cwl-red}
## Resources
If you’re interested in joining the CWL community, we recommend checking out the following resources and forums:

> * The place to ask a question about all things CWL is on [the CWL Discourse Group](https://cwl.discourse.group/). Previously we used [biostars.org](https://www.biostars.org/t/cwl/) where you can still read older responses.

Reviewer: ```suggestion
* The place to ask a question about all things CWL is [the CWL Discourse Group](https://cwl.discourse.group/). Previously we used [biostars.org](https://www.biostars.org/t/cwl/) where you can still read older responses.
```

> * If you are interested in learning more or contributing ideas or code, [come chat with us on Gitter](https://gitter.im/common-workflow-language/common-workflow-language), check out [#CommonWL on Twitter](https://twitter.com/search?q=%23CommonWL), join the [mailing list common-workflow-language on Google Groups](https://groups.google.com/forum/#!forum/common-workflow-language) or [fork the repository](https://github.com/common-workflow-language/common-workflow-language) and send a pull request!

Reviewer: ```suggestion
* If you are interested in learning more or contributing with ideas or code, [come chat with us on Gitter](https://gitter.im/common-workflow-language/common-workflow-language), check out [#CommonWL on Twitter](https://twitter.com/search?q=%23CommonWL), join the [mailing list common-workflow-language on Google Groups](https://groups.google.com/forum/#!forum/common-workflow-language) or [fork the repository](https://github.com/common-workflow-language/common-workflow-language) and send a pull request!
```

* Besides the web interface for the mailing list, one can also join by sending a blank email to [common-workflow-language+subscribe@googlegroups.com](mailto:common-workflow-language+subscribe@googlegroups.com) and replying to the automated response.
* If you would like to receive Google Calendar invites for the CWL weekly video chat, sign up at [https://groups.google.com/g/common-workflow-language-videochat-invites](https://groups.google.com/g/common-workflow-language-videochat-invites).

> The CWL Project is dedicated to providing a harassment-free experience for everyone, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, age, race, or religion. We do not tolerate harassment of participants in any form. This code of conduct applies to all CWL Project spaces, including the Google Group, the Gitter chat room, the Google Hangouts chats, both online and off. Anyone who violates this code of conduct may be sanctioned or expelled from these spaces at the discretion of the leadership team.

Reviewer: ```suggestion
The CWL Project is dedicated to providing a harassment-free experience for everyone, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, age, race, or religion. We do not tolerate harassment of participants in any form. This code of conduct applies to all CWL Project spaces, including the Google Group, the Gitter chat room, the Google Meet and Jitsi chats, both online and offline. Anyone who violates this code of conduct may be sanctioned or expelled from these spaces at the discretion of the leadership team.
```

Author: We don't use Google Meet anymore, so just Jitsi is okay here.

> For more details, see our [Code of Conduct](https://github.com/common-workflow-language/common-workflow-language/blob/main/CODE_OF_CONDUCT.md).

Author: ```suggestion
For more details, see our [Code of Conduct](/code-of-conduct/).
```
